semih2.js

```diff
@@ -0,0 +1,190 @@
+// #1
+let myString = "hello,this,is,a,difficult,to,read,sentence";
+console.log(myString);
+console.log(myString.length);
+
+const newString = myString.split(",").join(" ");
+console.log(newString);
+
+
+// #2
+let favoriteAnimals = ["blowfish", "capricorn", "giraffe"];
+favoriteAnimals.push("turtle");
+console.log(favoriteAnimals);
+
+favoriteAnimals.unshift("meerkat");
```

Comment:
unshift adds the value to the beginning of the array, but it needs to go after blowfish. You can use `splice` for this.

Reply:
Right, I misread it somehow and thought I should add to the beginning. I should have used `favoriteAnimals.splice(1, 0, `'meerkat');``